libs/pages/elewa/about-us/src/lib/components/history-carousel/history-carousel.component.html

```diff
@@ -0,0 +1,18 @@
+<div class="wrapper">
+    <div class="counter-col">
+        <h2>{{peoplecount}}+</h2>
+        <p class="count-text ">People trained</p>
+    </div>
+    <div class="counter-col">
+        <h2>{{careercount}}k</h2>
+        <p class="count-text ">Careers launched</p>
+    </div>
+    <div class="counter-col">
+        <h2>{{projectcount}}+</h2>
+        <p class="count-text ">Projects finished</p>
+    </div>
+    <div class="counter-col">
+        <h2>{{investorcount}} </h2>
+        <p class="count-text ">Investors</p>
+    </div>
```

Comment:
This is a completely different issue and also the component name does not match the contents and vice versa.